src/components/mgt-person/mgt-person.ts

```diff
@@ -457,6 +619,30 @@ export class MgtPerson extends MgtTemplatedComponent {
     }
 
     const graph = provider.graph.forComponent(this);
+    const betaGraph = BetaGraph.fromGraph(graph);
+
+    const defaultPresence = {
+      activity: 'Offline',
+      availability: 'Offline',
+      id: null
+    };
+
+    // populate presence
+    if (!this.personPresence) {
+      try {
+        if (this.userId) {
```

Comment:
We should be able to get presence even if using person query - similar to how we get the image. The presence can also be batched together with the image call for better performance.

Reply:
Yeah looking at the documentation, seemed like only id would work. Tried upn and email, both can be passed in to the {userid} bracket so will update this.

Reply:
Actually, after some testing... It seems like the json response will always indicate presence unknown if you pass in a query.

Reply:
I would first fetch the person (with `person-query` or `user-id`, etc). Once you have the person, you can check if that person is a user (if it has upn or check the type). If it's a user, fetch the presence using the id.

Reply:
Makes sense. All done

Reply:
I think you did this in the `mgt-people` but I still need to do this here (in `mgt-person`)